test/integration/patients/worklist/worklist.js

```diff
@@ -848,6 +1129,445 @@ context('worklist page', function() {
       .go('back');
   });
 
+  specify('action list - socket notifications', function() {
+    localStorage.setItem(`owned-by_11111_${ workspaceOne.id }-${ STATE_VERSION }`, JSON.stringify({
+      id: 'owned-by',
+      listType: 'actions',
+      actionsSortId: 'sortCreatedDesc',
+      clinicianId: '11111',
+      customFilters: {},
+      actionsSelected: {},
+      flowsSelected: {},
+    }));
+
+    const testComment = getComment();
+    const testSocketFileId = uuid();
+
+    const testFlow = getFlow({
+      attributes: {
+        name: 'Test Flow',
+      },
+      relationships: {
+        state: getRelationship(stateInProgress),
+      },
+    });
+
+    const testSocketAction = getAction({
+      attributes: {
+        name: 'Test Action - subscribed on page load',
+        details: '',
+        due_date: testDate(),
+        due_time: '06:00:00',
+        duration: 10,
+        updated_at: testTsSubtract(4),
+        created_at: testTsSubtract(4),
+      },
+      relationships: {
+        state: getRelationship(stateTodo),
+        flow: getRelationship(testFlow),
+        files: getRelationship([]),
```

Comment:
it's ok to do so, but I suspect we don't need to set a lot of this for the tests?  details?  due_time? duration. files relationship?  If they're just random values from `getAction` does that potentially cause flake?  Ideally were setting whatever minimum test data we need and letting `getAction` do the rest

Reply:
Agreed, I cleaned up a number of these: https://github.com/RoundingWell/care-ops-frontend/compare/003a41f45cc3b2abdda8b3aa33189122edb965a0..54708cca95f3c34880bfa2975ca936e7e887260c